### ISSUE TITLE
dynamic parkreferences

### ISSUE THREAD

**Issue by moep0710** [stijnwop/hoseSystem#65]
when parking a hose it should be possible to store it in another way than just straight.

Maybe defining 5 Points in the vehicle xml to adjust the hoses to.

Workaround: Adding a hose-mesh to your vehicle and toggle visibility when adding the hose

**Comment by stijnwop** [comment]
Euhm, perhaps you have not really looked around that much. But you can park the hose however you want.. so with curves.

Look at the kaweco DTS for example:  https://www.farming-simulator.com/mod.php?lang=en&country=nl&mod_id=104352&title=fs2017
It's def not the only mod around.

Also adding a replacement mesh is inefficient.

**Comment by stijnwop** [comment]
Also, if you looked at the feature list you would have noticed.

- Dynamic parking position [only define the start parking node and set an offset if needed.. the script will do the rest]
If applicable you could use animation on the park function for moving some arms/levers around

**Comment by moep0710** [comment]
Yeah, I should have showed you what I mean or describe it better. Here are 2 Screenshots:

1. Like it should be
2. like it is

I think using Offset does not support something like I'm interested in.
![screenshot 697](https://user-images.githubusercontent.com/43494277/46886053-08834800-ce5a-11e8-898f-1a0eaab3384a.png)
![screenshot 696](https://user-images.githubusercontent.com/43494277/46886054-08834800-ce5a-11e8-8fbd-b8abf471e599.png)

**Comment by moep0710** [comment]
It would be much easier to define a start and endNode and one middleNode (used by component1 to align).

4 meter
![screenshot 698](https://user-images.githubusercontent.com/43494277/46886976-b7c11e80-ce5c-11e8-8644-e240c690b98d.png)
3 meter :(
![screenshot 699](https://user-images.githubusercontent.com/43494277/46886977-b7c11e80-ce5c-11e8-905e-9f9187e233ad.png)
